lbz/resource.py

```diff
@@ -239,6 +239,7 @@ class EventAwareResource(Resource):
     def __init__(self, event: dict):
         super().__init__(event)
         self.event_api = EventAPI()
+        self.event_api.clear()
```

Comment:
I have exactly the same comment here, I don't think we should clear all the lists if the response is not OK 🤔 

The `clear()` method should be used only in two places OR nowhere if we are fully confident that our Singleton works correctly in Lambda functions.